scripts/reference-tables.js

```diff
@@ -321,8 +324,14 @@ addLandmarkRole(['region'], true, 'Region Landmark', 'landmarks/region.html');
 addLandmarkRole(['search'], true, 'Search Landmark', 'landmarks/search.html');
 
 function exampleListItem(item) {
+  let highContrast = '';
+  let title = '';
+  if (item.highContrast) {
+    highContrast = ' (<abbr title="High Contrast Support">HC</abbr>)';
+    title = 'High Contrast Support';
+  }
   return `
-                <li><a href="${item.ref}">${item.title}</a></li>`;
+                <li><a href="${item.ref}" title="${title}">${item.title}</a>${highContrast}</li>`;
```

Comment:
The empty `title` issue could be fixed if the `title="` string were part of the variable, e.g.:

```
let title='';
(...)
title='title="High Contrast Support"`;
(...)
<a href="etc"${title}>
```

Though I still think the link tooltip is better left off entirely :D